src/components/SignIn/SignInScreen.js

```diff
@@ -1,5 +1,101 @@
-import React from 'react';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import React, { useMemo, useState } from 'react';
+import './SignIn.css';
+import Airlock from '@calblueprint/airlock';
+
+Airlock.configure({
+  endpointUrl: 'http://localhost:3000',
+  apiKey: 'airlock',
+});
+const BASE_ID = process.env.REACT_APP_AIRTABLE_BASE_ID;
+
+const base = Airlock.base(BASE_ID);
 
 export default function SignInScreen() {
-  return <h1>SignIn Screen</h1>;
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [errorMsg, setErrorMsg] = useState(null);
+  const [userName, setUserName] = useState('');
+  const onSubmit = async (evt) => {
+    evt.preventDefault();
+    setLoading(true);
+    try {
+      const login = await base.login({ username: email, password });
+      const { body: { user } } = login;
+      setUserName(`Welcome ${user.fields.display_name}!`);
```

Comment:
Ooooh this is what you mean't by the welcome message! This is a nice touch it was really fun to interact with haha